syntax = "proto3";

package location;

message LocationRequest {
  int32 location_id = 1;
}

message LocationResponse {
  int64 id = 1;
  int32 person_id = 2;
  string latitude = 3;
  string longitude = 4;
  string creation_time = 5;
}

service LocationService {
  rpc GetLocation (LocationRequest) returns (LocationResponse);
}
